common/lib/capa/capa/templates/formulaequationinput.html

```diff
@@ -4,15 +4,13 @@
     <input type="text" name="input_${id}" id="input_${id}"
         data-input-id="${id}" value="${value|h}"
         aria-label="${label}"
+        aria-describedby="${id}_mathjax"
```

Comment:
@ahsan-ul-haq can we add an ID to the ```<p class="status">``` below and reference that in the aria-describedby? (it takes a space separated list of IDs:
```aria-describedby="${id}_mathjax ${id}_status"```